data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -5,7 +5,8 @@ set -euoE pipefail ## -E option will cause functions to inherit trap
 
 mkdir --parents /etc/kubernetes/{manifests,bootstrap-configs,bootstrap-manifests}
 
-ETCD_ENDPOINTS={{.EtcdCluster}}
+BOOTSTRAP_IP=$(hostname -I | awk '{ print $1 }')
+ETCD_ENDPOINTS=https://"${BOOTSTRAP_IP}":2379
```

Comment:
Until CEO is ready, can we make sure this value is known empty/broken by default so that the error is clear. And set the correct value in the if else?

Reply:
I left ETCD_ENDPOINTS populated with a known value. Then added an additional if block[1] to handle retry.

[1] https://github.com/openshift/installer/pull/2767/files#diff-ce82c1d8a44f7dfc41dfc024085ccfeeR118